src/index.html

```diff
@@ -7,8 +7,68 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 hidden>Moyo header</h1>
+
+    <header class="header">
+      <a class="logo" href="#home">
+        <img
+        src="./images/logo.png"
+        alt="MOYO_logo"
+      >
+      </a>
+      <nav class="nav">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
```
fix the code style all over the page